src/test/java/harness/utils/WorkspaceUtils.java

```diff
@@ -0,0 +1,48 @@
+package harness.utils;
+
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import harness.CRLJanitor;
+import harness.TestCommand;
+import harness.TestUser;
+
+/** Utilities for working with workspaces in CLI tests. */
+public class WorkspaceUtils {
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace and should be used to clean it up.
+   */
+  public static UFWorkspace createWorkspace(TestUser workspaceCreator)
+      throws JsonProcessingException {
+    // `terra workspace create --format=json`
+    UFWorkspace workspace =
+        TestCommand.runAndParseCommandExpectSuccess(UFWorkspace.class, "workspace", "create");
+    CRLJanitor.registerWorkspaceForCleanupIfEnabled(workspace, workspaceCreator);
+    return workspace;
+  }
+
+  /**
+   * Create a new workspace and register it with Janitor if this test is running in an environment
+   * where Janitor is enabled.
+   *
+   * @param workspaceCreator The user who owns the workspace and should be used to clean it up.
```

Comment:
opt My personal preference is to omit these `@param` javadocs. go/java-style says `Javadoc is optional for "simple, obvious" members`. Something like

```
* @param name The name of the workspace
```

isn't adding anything.

Reply:
Good point. I think it's worth documenting that `workspaceCreator` will be impersonated in a WSM call here, but the other params aren't adding anything.